db/migrate/20170918190738_issue_shorback_tiers.rb

```diff
@@ -0,0 +1,13 @@
+class IssueShorbackTiers < ActiveRecord::Migration[5.0]
+  def up
+    rename_column    :showback_tiers, :showback_rates_id, :showback_rate_id
+    change_column    :showback_tiers, :tier_start_value,  :float
+    change_column    :showback_tiers, :tier_end_value,    :float
```

Comment:
To be clear, the current chargeback uses floats and we mistakingly are using bigint here.  We only really want to use Float for Float::INFINITY.